src/components/BusInfoCarousel.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Slider from 'react-slick';
+import BusInfoContainer from './BusInfoContainer';
+import { IoIosArrowForward, IoIosArrowBack } from 'react-icons/io';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+export default class BusInfoCarousel extends Component {
+  constructor(props) {
+    super(props);
+    this.next = this.next.bind(this);
+    this.previous = this.previous.bind(this);
+    this.onKeyDown = this.onKeyDown.bind(this);
+  }
+
+  next() {
+    this.slider.slickNext();
+  }
+
+  previous() {
+    this.slider.slickPrev();
+  }
+
+  onKeyDown(event) {
```

Comment:
this will have to move to the page, because currently, it works only if the bus container is in focus.